src/env_access.c

```diff
@@ -0,0 +1,16 @@
+#include "env_access.h"
+
+char	*env_get_user(t_env *_env)
+{
+	return (env_get(_env, "$USER"));
+}
+
+char	*env_get_host(t_env *_env)
+{
+	return (env_get(_env, "$HOST"));
+}
+
+char	*env_get_working_dir(t_env *_env)
+{
+	return (env_get(_env, "$PWD"));
+}
```

Comment:
remove the $ in front of variables, the $ is only a signifier in the bash language to signal that the word attached is a variable name. the dollar itself is not part of the key.